synapse/models/person.py

```diff
@@ -60,9 +58,11 @@ def getModelDefs(self):
                     'doc': 'A GUID for a list of associated contacts.',
                 }),
                 ('ps:workhist', ('guid', {}), {
-                    'doc': "A GUID representing entry in a contact's work history.",
-                }),
+                    'doc': "A GUID representing entry in a contact's work history."}),
```

Comment:
should this be "an entry" or is shortening it to "entry" fine?

Reply:
haha, collateral damage from adjusting whitespace. We've also moved away from "A GUID representing" so, i'll update to "An entry in a..." :+1: